doc/release-notes/rl-2511.section.md

```diff
@@ -3,6 +3,9 @@
 ## Highlights {#sec-nixpkgs-release-25.11-highlights}
 <!-- To avoid merge conflicts, consider adding your item at an arbitrary place in the list instead. -->
 
+- The `x86_64-darwin` platform will be deprecated within the next few years, in light of Apple’s announcement that macOS 26 will be the final version to support Intel Macs.
+  Based on Apple’s security support policy and our available build and maintenance resources, we expect the first Nixpkgs release without `x86_64-darwin` support to be between 26.11 and 28.11.
```

Comment:
Realistically, I don’t think we’ll be able to maintain support past 27.05. 27.11 will release after macOS 28, which will have diminished Rosetta 2 support. Depending on what that looks like, it may not be feasible for contributors to build and test on x86_64-darwin. We could maintain x86_64-darwin as a cross target, but that comes with some pretty big caveats (like no Python support).